### PR TITLE
Convox Tips: Disabling racks on evenings and weekends

### PR DESCRIPTION
**Pull request by mwarkentin**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by nzoschke** [comment]
Great content. Thanks for writing it up!

**Comment by mwarkentin** [comment]
Have a couple of tweaks yet. 😊

**Comment by soulshake** [comment]
I tested it out locally and it seems you need to put your title in quotes since it contains a colon. #yamlproblems

Maybe we can wait a day or two and then update the date of the post, so it doesn't immediately get bumped off the top by the one @mattmanning published today :)

**Comment by mattmanning** [comment]
@soulshake @nzoschke Thanks for reviewing.

I've been talking to @mwarkentin in Slack about the post. I'm gonna help get it formatted for publishing hopefully Monday.

**Comment by mwarkentin** [comment]
* Can you set future dates for posts? Looks like it doesn't show up on the site locally if I set the date to tomorrow?

File: _posts/2017-01-09-convox-tips-disabling-racks-evenings-weekends.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+Find your Auto Scaling Group (called `convox-<rack>-Instances-<id>`), and switch to the Scheduled Actions tab:
+
+![](https://cdn-images-1.medium.com/max/2000/1*O-eoQtSzpTpIbkli_y_gTg.png)
+
+Create the Scale Down action by clicking on the Create Scheduled Action button, and fill in the form:
+
+![](https://cdn-images-1.medium.com/max/2000/1*pNh_WBr3rAoq-Ufzzl6I_A.png)
+
+Make sure to adjust the cron schedule to fit your requirements — and make sure you convert the times to UTC
```
Comment (mwarkentin) [inline _posts/2017-01-09-convox-tips-disabling-racks-evenings-weekends.md]: Is there a way to do image captions for this like I have on the medium post?

File: _posts/2017-01-08-convox-tips-disabling-racks-evenings-weekends.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Find your Auto Scaling Group (called `convox-<rack>-Instances-<id>`), and switch to the Scheduled Actions tab:
+
+![](https://cdn-images-1.medium.com/max/2000/1*O-eoQtSzpTpIbkli_y_gTg.png)
+
+Create the Scale Down action by clicking on the Create Scheduled Action button, and fill in the form:
+
+![](https://cdn-images-1.medium.com/max/2000/1*pNh_WBr3rAoq-Ufzzl6I_A.png)
+
+Make sure to adjust the cron schedule to fit your requirements — and make sure you convert the times to UTC
```
Comment (mwarkentin) [inline _posts/2017-01-08-convox-tips-disabling-racks-evenings-weekends.md]: Is there a way to do image captions like I have on the medium post?

File: _posts/2017-01-08-convox-tips-disabling-racks-evenings-weekends.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+## Final Notes
+
+* Sometimes, a dev needs to do some work late in the evening, or on a weekend. We’ve created Jenkins jobs to allow our devs to turn the environment on/off as needed. We also added a second scale down time late at night in case the developer forgot to turn the rack back off.
+* It usually takes between 5–10 minutes for a rack to spin up and start all of the app processes.
+* We’re saving over $200/m by scaling our dev racks down during off hours. This will only grow as we increase instance sizes, counts and number of developer racks.
+
+## About Wave
+
+[Wave](https://www.waveapps.com/) provides invoicing, credit card payments, accounting, and payroll for small businesses. If you're interested in joining the team, check out our [careers page](https://www.waveapps.com/about-us/jobs/) and get in touch!
```
Comment (mwarkentin) [inline _posts/2017-01-08-convox-tips-disabling-racks-evenings-weekends.md]: @dlanger Let me know if we have any sort of usual blurb we use for stuff like this or if you have any other suggestions for what to drop here.

Comment (dlanger) [inline _posts/2017-01-08-convox-tips-disabling-racks-evenings-weekends.md]: The standard one for pieces where we're advocating for Wave as a company (as opposed to the product) is something like this:

_Composed of some of the smartest minds in North America — more than 100 and counting — the Wave team is innovative, diverse, driven, and incredibly talented. They're also some of the nicest, most interesting, creative, and inspiring people around. Join us at [waveapps.com/careers](http://waveapps.com/careers)._

I'd start with that, and tone it up/down as you're comfortable with.

**Comment by mwarkentin** [comment]
@soulshake @mattmanning Ok, ready for final review and publishing if things look good to you. :)

Let me know if you want the branch rebased into a single commit.

**Comment by soulshake** [comment]
[Publish mwarkentin blog post](https://trello.com/c/g7r5QUAw/1504-publish-mwarkentin-blog-post)

File: _posts/2017-01-09-convox-tips-disabling-racks-evenings-weekends.md (hunk 1 above)
Comment (soulshake) [inline _posts/2017-01-09-convox-tips-disabling-racks-evenings-weekends.md]: I just opened PR #438 to facilitate captions. Once it's merged you'll be able to add captions by putting `*Some text between asterisks*` just after the image on the same line.

**Comment by mwarkentin** [comment]
Verified image caption styling looks good locally:

**Comment by mwarkentin** [comment]
@soulshake ^^

**Comment by mattmanning** [comment]
@mwarkentin Thanks for adding the captions. I was just about to do that :)

This looks good to me. Can I ask you to just update the filename to use 2017-01-10? I'll merge and publish this tomorrow morning. Don't want it to get buried at the end of the day today.

Thanks again for the PR.

**Comment by mwarkentin** [comment]
Will do.

**Comment by mwarkentin** [comment]
@mattmanning Done!

**Pull request merged**